lib/api/Lookupd.js

```diff
@@ -2,7 +2,7 @@
 
 const Base = require('./Base');
 
-module.exports = class Nsqlookupd extends Base {
+module.exports = class Lookupd extends Base {
```

Comment:
Damn, this is a breaking change actually.

Reply:
:see_no_evil: